Item/Item.module.less

```diff
@@ -95,6 +95,8 @@
 
 		.focus({
 			border-color: @agate-item-focus-border-color;
+			border-image-slice: 1;
```

Comment:
This affects other skins. Let's define 1 as a variable in variables-base.less and variables-silicon.less

/* gallium skin */
```
.Item_item__1OGdw.gallium.spottable :focus {
    ...
    border-image-slice: 1;
    ...
}
```